test/Microsoft.AspNet.Server.KestrelTests/Program.cs

```diff
@@ -8,9 +11,21 @@ namespace Microsoft.AspNet.Server.KestrelTests
     /// </summary>
     public class Program
     {
-        public void Main()
+        private readonly IApplicationEnvironment env;
+        private readonly IServiceProvider sp;
+
+        public Program(IApplicationEnvironment env, IServiceProvider sp)
+        {
+            this.env = env;
+            this.sp = sp;
+        }
+
+        public int Main()
         {
-            new EngineTests().DisconnectingClient().Wait();
+            return new Xunit.Runner.AspNet.Program(env, sp).Main(new string[] {
+                "-class",
```

Comment:
Might be nice to take an argument for the test class name since this seems pretty useful in general.